Backend/src/main/java/com/abhinternship/auctionapp/model/AuctionNotification.java

```diff
@@ -0,0 +1,40 @@
+package com.abhinternship.auctionapp.model;
+
+public class AuctionNotification {
```

Comment:
AuctionNotification and BidNotification have properties in common. Think about making a notification base class.